Update vnet_service_endpoint_overview_armaha (#63364)
Azure/azure-content: articles/azure-sql/database/vnet-service-endpoint-rule-overview.md

- Turning ON virtual network service endpoints to Azure SQL Database also enables the endpoints for the MySQL and PostgreSQL Azure services. However, with endpoints ON, attempts to connect from the endpoints to your MySQL or PostgreSQL instances may fail.
  - The underlying reason is that MySQL and PostgreSQL likely do not have a virtual network rule configured. You must configure a virtual network rule for Azure Database for MySQL and PostgreSQL and the connection will succeed.
  

> - If Private endpoints are enabled for the SQL server already, then the Deny public network access should be set to No , reason being the Vnet service endpoints for Azure SQL listens on public endpoints and should be allowed in order to add vnet rules to the SQL server firewall.

Reviewer: Hey @aravindmahadevan-ms - Thanks for submitting this. Per @rohitnayakmsft, please change the text to the following, which will give you the correct styling:

```suggestion
- To define VNet firewall rules on a SQL logical server that is already configured with private endpoints, set **Deny public network access** to **No**.
```
